src/main/java/org/zalando/nakadi/domain/EventType.java

```diff
@@ -74,6 +80,8 @@ public EventType(final String name, final String topic, final String owningAppli
         this.schema = schema;
         this.defaultStatistic = defaultStatistic;
         this.options = options;
+        this.writeScope = writeScope;
+        this.readScope = readScope;
```

Comment:
I think I don't understand that. If the name was `readScopeSet` - that would be more or less fine. Read scope identifies that it's a single one.